### ISSUE TITLE
[Bug]: after updating to v26 the entire themeing/design is missing, making it unusable

### ISSUE THREAD

**Issue by elandorr** [nextcloud/server#37777]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

see title
looks like this 
![image](https://user-images.githubusercontent.com/56206745/232624875-27742777-633d-447d-9f20-b7bc65de8db8.png)

No errors in console besides a whole bunch of deprecations in JS stuff NC uses.

I thought the days of OC/NC dying every update were over. Apparently not.

Anyone got an idea? I'd absolutely hate having to deal with restoring a backup.

### Steps to reproduce

1. update
2. open eyes

### Expected behavior

a working instance

### Installation method

Community Manual installation with Archive

### Nextcloud Server version

26

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Nginx

### Database engine version

None

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
no, contains private data

only useful thing would be this anyway:

26.0.0.11
```

### List of activated Apps

```shell
Enabled:                                                                                                  
  - activity: 2.18.0                                                                                      
  - cloud_federation_api: 1.9.0
  - contactsinteraction: 1.7.0
  - dashboard: 7.6.0
  - dav: 1.25.0
  - federatedfilesharing: 1.16.0
  - files: 1.21.1
  - files_pdfviewer: 2.7.0 
  - files_rightclick: 1.5.0
  - files_sharing: 1.18.0
  - logreader: 2.11.0
  - lookup_server_connector: 1.14.0
  - nextcloud_announcements: 1.15.0
  - notifications: 2.14.0
  - oauth2: 1.14.0
  - password_policy: 1.16.0
  - privacy: 1.10.0
  - provisioning_api: 1.16.0
  - related_resources: 1.1.0-alpha1
  - serverinfo: 1.16.0
  - settings: 1.8.0
  - text: 3.7.2
  - theming: 2.1.1
  - twofactor_backupcodes: 1.15.0
  - updatenotification: 1.16.0
  - viewer: 1.10.0
  - workflowengine: 2.8.0
```

### Nextcloud Logs

```shell
No. It's huge and I don't have time to check for private data.
```

**Comment by elandorr** [comment]
You labeled this 'stable'. Seriously. I hesitated and trusted it would work this time.

There are no logs anywhere. No assets that 404, no PHP errors, nothing. I don't even know where to begin debugging this mess.

**Comment by joshtrichards** [comment]
I'm unable to reproduce this, but there are some other things in the mix in your environment. I'm also a bit suspicious something else is going on if you're seeing issues like that in your restored deployment.

Nextcloud is like any other web app - and even if NC was completely hosed your own web server has log files that are outside the purview of NC's own logging where errors would still be showing up.

This sounds a lot like a permissions/file ownership issue or `location` config issue in your web server config. It could also just be something missed along the way during a prior update/upgrade - if you're truly doing it manually - that has cascaded.

* Can you confirm the issue occurs in more than one browser?
* Are you certain no browser extensions are blocking anything?
* Is your web server configured to log all requests? What do your web server logs show? 

Are you certain there are no other messages in the browser console indicating 403, 500, etc.? In Chrome they generally show up in red versus the deprecated messages which are orange/yellow (and I only see ~7 of those on the login page)

I see you stated you're using nginx as your main web server - which isn't officially supported. Can you provide some details on the nature of your nginx configuration?

Please check this too: https://docs.nextcloud.com/server/stable/admin_manual/installation/nginx.html#javascript-js-or-css-css-files-not-served-properly

Lastly, is there a reverse proxy or anything of the like in front of this web server? If so, its logs (and logging configuration) should be checked as well.

Oh, how did you upgrade? https://docs.nextcloud.com/server/stable/admin_manual/maintenance/upgrade.html - The built-in updater or manually upgrading via the tar.gz archive? i.e. https://docs.nextcloud.com/server/stable/admin_manual/maintenance/update.html or https://docs.nextcloud.com/server/stable/admin_manual/maintenance/manual_upgrade.html?

**Comment by c-pfuhl** [comment]
I have only a very small part of the problem in the login screenshot, the overflowing account name and password fields (looks exactly the same for me with the different lengths). 

This started after the upgrade to NC 25 and still appears after the upgrade to 26. The only browser of the ones i tested this on where this happens is qutebrowser which is using qtwebengine based on chromium 87.0.4280.144. No problems with firefox or chromium 110. So i guess this is an "old browser engine" problem, i also get an "unsupported browser" warning since NC 26. There don't seem to be any warnings or errors related to the problem in nextcloud.log. Everything that appears when the login page is opened also appears on firefox. Also nothing in nginx error.log.

I can make the login form look pretty much like in firefox if i change the marked padding values of 28px and 12px in the browser devtools styles settings to 1px each
![loginform](https://user-images.githubusercontent.com/131558091/233808253-0ab2015c-31c7-412c-bb73-d24797e81b7a.png)

**Comment by elandorr** [comment]
> own web server has log files that are outside the purview of NC's own logging where errors would still be showing up.

No wrong/unusual requests were made, the web server can't log what doesn't exist.

> Can you confirm the issue occurs in more than one browser?

I tried our custom FF + regular TBB, it didn't seem to be a browser issue.

> Are you certain no browser extensions are blocking anything?

Yes, as TBB has nothing (except noscript).

> Is your web server configured to log all requests? What do your web server logs show?

Temporarily only, yes, but nothing except 200s all the way. NC causes a few too many requests to go one by one and wonder what might've happened, unfortunately. But they were all fine, no errors.

> Are you certain there are no other messages in the browser console indicating 403, 500, etc.?

Yes, that was the most annoying part, as I had no indication where to even start looking.

> Can you provide some details on the nature of your nginx configuration?

Using 99% the config from the docs (which may not be 'official' and 'only' user-written, but has been used for years).

> Lastly, is there a reverse proxy or anything of the like in front of this web server? If so, its logs (and logging configuration) should be checked as well.

Nope

> The built-in updater or manually

Via occ as usual.

I kept things simple:

- uploaded the mysteriously disappearing logo again
- vowed to never update to a .0.0 again, backups are painful without a bigger budget 

I'm sorry, I don't have time to debug for a multi-million $ business for free. I work on non-profit stuff and already lack a budget. I almost guarantee it's like usual: Some update down the line will be fine, nobody will know why, and in a year I'll be stupid enough to update first yet again. We've had a few good updates without issues, it had to happen.

Never be the first to push the button, my friends.

I'll close this as I can't provide data. If @c-pfuhl likes to debug the small visual fix that may or may not be related, you can still re-open.

> an "old browser engine" problem

For the record: I only use the regular ESR as Debian thankfully does not rush for beta-quality releases. TBB = FF ESR as well. But the ESR is far from outdated, that'd be weird, if NC used some hyper-new CSS.

@joshtrichards Sorry for the tone, I just had flashbacks to the days when I had to re-install OC every frickin' time because it'd inevitably kill itself. How about you make a 'boring' spin-off that updates like once every 5 years (except for security patches)?

I'm willing to bet most users would be fine with a 'boring' edition. Heck, if Mozilla and Google hadn't murdered FTP, that'd still be fine for most use cases. A few keystrokes to upload something, downloads working in every browser/OS without a dozen + MB of JS.

That's kind of a big deal, by the way. Given the economic disasters, people try to save even bandwidth. NC is gigantic.

![image](https://user-images.githubusercontent.com/56206745/235312535-57f27162-d6db-4ed1-89e7-4d3c47fa4586.png)

![image](https://user-images.githubusercontent.com/56206745/235312539-25afebcf-8424-4010-a9f4-fcec03111a9b.png)

NC is officially bigger than a fully functional Linux distro. :) (and our config is already minimal with no apps enabled)

**Comment by AlexanderStein** [comment]
Unfortunately this issue was closed. I'm hit by the exact same problem. No errors in the logs or whatsoever. Also the browser sucessfully loads all files, 200 HTTP codes. I can understand that frustration. I even checked the checksums for the .js and .css files the browser downloaded, looks good so far. But I was not able to track down `apps/theming/theme/light-highcontrast.css` and similar. Apparently they are generated on the fly.

Logging into nextcloud actually works, but it is hard or impossible to access several things as the layout and fonts are disturbed.

As the backend apparently works (uploading files from nextcloud android app works seemingly), it seems to be a frontend/css/theming problem. I tried disabling the apcu cache, browser caches, including forced reloads. Nothing.

**Comment by elandorr** [comment]
don't know how github permissions work, but as I can reopen, let me do that for you real quick

before you go mad I'd consider doing what I did and just 'waiting it out' until a couple .x releases in

**Comment by AlexanderStein** [comment]
I was able to fix this on my machine. I tried to reinstall nextcloud, but even the landing page for new installations had this issue, so it's unrelated to nextcloud.

Instead I had to update the Nginx site config file according to https://docs.nextcloud.com/server/latest/admin_manual/installation/nginx.html#nextcloud-in-a-subdir-of-the-nginx-webroot. A few config options regarding were different and need to be adjusted. Once changed the theme was showing properly immediately
